pom.xml

```diff
@@ -74,18 +139,56 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>script-security</artifactId>
+      <version>1.78</version>
+    </dependency>
```

Comment:
You might just want to resolve `RequireUpperBoundDeps`.
It's not good way to add explicit dependencies to resolve that.
Following documents may help you:
* https://www.jenkins.io/doc/developer/plugin-development/dependency-management/
* https://github.com/jenkins-infra/docker-confluence-data/blob/a336b7af88e56d32e11fce0aac1f966f74e279c5/content/JENKINS/How-to-fix-RequireUpperBoundDeps.html

Targetting LTS core and using BOM is the easiest way.